lib/fireblocks/api/transactions.rb

```diff
@@ -24,58 +27,81 @@ def create(options)
         end
 
         def from_vault_to_external(
-          amount:,
+          amount: nil,
```

Comment:
Not 100% sure, but we may want this to be the sum of all destinations. That's how I tested the API call. I didn't try it without this summed amount